### DISCUSSION TITLE
huh, neat

### DISCUSSION THREAD

**Discussion by signupisnonsense** [cthulhustig/autojimmy#12] [General]
Looks like my account is still active...

So, stuck with debian 11 and python 3.9 for the foreseeable future.  Have also had issues with pip and apt in the past.  Gave this a shot, and here's some notes on getting it run:

apt-cache found version 5.15 of PyQt and PyQtWebEngine, 0.11 of pyqtgraph, and 3.5.50 of reportgraph.  Manually installed 0.7.1 of darkdetect (there is a new version, didn't try it).  Started it up and slammed into issues with using '|' instead of 'or' in:
gui/trader/localbrokerwidget.py and
gui/trader/sharedconfigwidgets.py
Changed those out and it runs fine.  I think support for '|' happened in python 3.10 (not a python guy, so no real idea).

**Comment by cthulhustig** [comment]
Thanks for letting me know. I think this should have been fixed by a change I pushed a few days ago (PR #3). If your still seeing it when using the tip of main could you let me know.

**Comment by signupisnonsense** [comment]
All good.  Not sure when I did the first pull.
